ui/pages/confirmations/components/confirm/info/typed-sign/typed-sign.tsx

```diff
@@ -81,6 +82,13 @@ const TypedSignInfo: React.FC = () => {
             />
           </ConfirmInfoRow>
         )}
+        <ConfirmInfoAlertRow
```

Comment:
Minor, could we create a single definition of this row for signatures to avoid the duplication?

Similar to transactions where we re-use standard row components.

Reply:
I did this re-factor